src/examples/mailchimp/src/backend/schema/member/provider.ts

```diff
@@ -0,0 +1,81 @@
+import Mailchimp from '@mailchimp/mailchimp_marketing/src/ApiClient';
+import {
+	createProvider,
+	ItemWithId,
+	createPaginationOptions,
+	inMemoryFilterFor,
+} from '@exogee/graphweaver-helpers';
+import { ClientOptions } from '../index';
+import { MemberDataEntity, MemberStatus } from './data-entity';
+import md5 from 'md5';
```

Comment:
I think we use argon2 for encryption. Can we double check that this is this lib to use with @steve-keep ?

Reply:
Is argon2 the same as md5? Mailchimp require md5 hashing on the email field.